dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/AbstractClouderaTimeSeriesTask.java

```diff
@@ -34,19 +35,31 @@
 abstract class AbstractClouderaTimeSeriesTask extends AbstractClouderaManagerTask {
   private static final DateTimeFormatter isoDateTimeFormatter =
       DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
-  private final ObjectMapper objectMapper = new ObjectMapper();
   private final int includedLastDays;
   private final TimeSeriesAggregation tsAggregation;
+  private final TaskCategory taskCategory;
 
   public AbstractClouderaTimeSeriesTask(
-      String targetPath, int includedLastDays, TimeSeriesAggregation tsAggregation) {
+      String targetPath,
+      int includedLastDays,
+      TimeSeriesAggregation tsAggregation,
+      TaskCategory taskCategory) {
     super(targetPath);
     Preconditions.checkNotNull(tsAggregation, "TimeSeriesAggregation has not to be a null.");
     Preconditions.checkArgument(
         includedLastDays >= 1,
         "The chart has to include at least one day. Received " + includedLastDays + " days.");
+    Preconditions.checkNotNull(taskCategory, "TaskCategory has not to be a null.");
```

Comment:
```suggestion
    Preconditions.checkNotNull(taskCategory, "TaskCategory must be not null.");
```

Reply:
Let's keep consistency 
https://github.com/google/dwh-migration-tools/pull/781/files/fbf1860287a0b36b1f6f3ea8a2b96abc0a42a08b#diff-1cc3adf020b1102c56090b1ad5f8d705417a7d79595984c4145419420a21e6b8R48

Reply:
Than it should be `has to be not a null` or `doesn't have to be a null`. I'm not sure that `has not to ...` be grammatically correct.

Reply:
fixed.